src/v/kafka/groups/group_router.h

```diff
@@ -39,13 +40,38 @@ namespace kafka {
  * the owning shard is found using the cluster::shard_table. Finally, a x-core
  * operation on the destination shard's group manager is invoked.
  */
-template<typename GroupMgr>
 class group_router final {
+    template<typename FwdFunc, typename Request>
```

Comment:
it's odd that you use the functions in reverse order of declaration.

Reply:
nit: i'd suggest re-ordering.

Reply:
oh yeh that is weird. will fix